plugin-infra/go-plugin-access/src/main/java/com/thoughtworks/go/plugin/access/elastic/ElasticAgentPluginRegistry.java

```diff
@@ -66,10 +66,10 @@ public String getPluginStatusReport(String pluginId) {
         return statusReportView;
     }
 
-    public String getAgentStatusReport(String pluginId, JobIdentifier jobIdentifier, String elasticAgentId) {
-        LOGGER.debug("Processing get plugin status report for plugin: {} with job-identifier: {} with elastic-agent-id: {}", pluginId, jobIdentifier, elasticAgentId);
-        final String agentStatusReportView = extension.getAgentStatusReport(pluginId, jobIdentifier, elasticAgentId);
-        LOGGER.debug("Done processing get plugin status report for plugin: {} with job-identifier: {} with elastic-agent-id: {} ", pluginId, jobIdentifier, elasticAgentId);
+    public String getAgentStatusReport(String pluginId, JobIdentifier jobIdentifier, String elasticAgentId, Map<String, String> clusterProfile) {
+        LOGGER.debug("Processing get plugin status report for plugin: {} with job-identifier: {} with elastic-agent-id: {} and cluster-profile: {}", pluginId, jobIdentifier, elasticAgentId, clusterProfile);
+        final String agentStatusReportView = extension.getAgentStatusReport(pluginId, jobIdentifier, elasticAgentId, clusterProfile);
+        LOGGER.debug("Done processing get plugin status report for plugin: {} with job-identifier: {} with elastic-agent-id: {} and cluster-profile: {}", pluginId, jobIdentifier, elasticAgentId, clusterProfile);
```

Comment:
Just for the consistency, can we say `cluster` as that's what we've been using everywhere?